docs/tutorials/essentials/part-3-data-flow.md

```diff
@@ -212,7 +232,7 @@ export type RootState = ReturnType<typeof store.getState>
 // highlight-end
 ```
 
-If you hover over the `RootState` type in your editor, you should see `type RootState = { value: number; }`. Since this type is automatically derived from the store definition, all the future changes to the `reducer` setup will automatically be reflected in the `RootState` type as well. This way we only need to define it once, and it will always be accurate.
+If you hover over the `RootState` type in your editor, you should see `type RootState = { counter: CounterState; }`. Since this type is automatically derived from the store definition, all the future changes to the `reducer` setup will automatically be reflected in the `RootState` type as well. This way we only need to define it once, and it will always be accurate.
```

Comment:
Please rename `RootState` to `AppRootState` to avoid people accidentally importing the wrong `RootState` type by accident when auto-importing and to be consistent with the other App specific redux names.

Reply:
We've been pretty consistent with using `RootState` in all of our documentation, so I'd prefer not to change it here. (Really, the better answer is to change the export from RTKQ, but we can't do that until the next major version.)